elections.md

```diff
@@ -65,40 +57,81 @@ representation is achieved. If it is impossible to find sufficient members to
+non-involved Steering Committee members will decide.
 
-### Initial Election
+### Terms and Election Cycles
+
+Steering committee members are elected to serve two year terms.
+
+Election cycles are scheduled such that roughly half of the seats come up for
+re-election each year for purposes of continuity.  The exact number of seats
+alternates between 3 and 4, with the first 3-seat election taking place in
+2018.
+
+## Emeritus Term
+
+Members of the steering committee will graduate to becoming Emeritus members of
+the steering committee upon vacating their seat.  This confers honor on the
+recipient, acknowledging the significant contributions they have made to the
+project. Emeritus members have no binding vote, and no expectation of continued
+participation in steering committee affairs.
 
-Because of the need to bootstrap a staggered election cycle, some of the
-initial committee members will only serve a single year term. These three
-people will be the lowest vote getters from the top six in the initial
-election.
+## Election schedule and operation
 
-The bootstrap committee will operate the election and circulate a timeline for
-nominations, and the vote.
+The steering committee picks election officers to operate the election and
+circulate a timeline for nominations, and the vote. At present because results
+are announced early October, the steering committee should start looking at
+this process by mid-July, planning to kick things off by mid-August. This
+process is articulated in [kubernetes/community/events/elections/README.md].
 
 ### Vacancies
 
-In the event of a resignation or other loss of a steering committee member, the
-candidate with the next most votes from the previous election will be offered
-the seat.  This process will continue until the seat is filled.  In case this
-fails to fill the seat, a special election for that position will be held as
-soon as possible. The same group of people as described in “eligibility for
-voting” will vote in the special election. A committee member elected in a
-special election will serve out the remainder of the term for the person they
-are replacing, regardless of the length of that remainder. If a continuity
-member resigns or otherwise is lost from the Committee, that position will not
-be filled.
+In the event of a resignation or other loss of a [bootstrap committee member],
+the position will not be refilled.
+
+In the event of a resignation or other loss of an elected steering committee
+member, the candidate with the next most votes from the previous election will
+be offered the seat.  This process will continue until the seat is filled.
+
+In case this fails to fill the seat, a special election for that position will
+be held as soon as possible. Eligible voters from the most recent election
+will vote in the special election (ie: eligibility will not be redetermined
+at the time of the special election). A committee member elected in a special
```

Comment:
Unsure if it's a big deal, but I changed special election wording to be explicit that we're not determining a new pool of voters for a special election at the time of a special election.